0009-simple-payment-setup-protocol/0009-simple-payment-setup-protocol.md

```diff
@@ -66,22 +71,25 @@ The SPSP Endpoint MUST respond to HTTPS `GET` requests in the following manner:
 The sender queries the SPSP endpoint to get information about the type of payment that can be made to this receiver:
 
 #### Request
+
 ``` http
 GET /api/spsp/bob HTTP/1.1
 Host: red.ilpdemo.org
-Accept: application/json
+Accept: application/spsp+json
 ```
 
 #### Response
 ``` http
 HTTP/1.1 200 OK
-Content-Type: application/json
+Content-Type: application/spsp+json
 
 {
   "destination_account": "example.ilpdemo.red.bob",
-  "shared_secret": "6jR5iNIVRvqeasJeCty6C-YB5X9FhSOUPCL_5nha5Vs",
-  "maximum_destination_amount": "100000",
-  "minimum_destination_amount": "10",
+  "shared_secret": "6jR5iNIVRvqeasJeCty6C+YB5X9FhSOUPCL/5nha5Vs=",
+  "balance": {
+    "maximum": "100000",
+    "current": "5360"
+  },
   "ledger_info": {
     "currency_code": "USD",
     "currency_scale": 2
```

Comment:
Let's remove these.

Reply:
I think it's still useful to have these optional available to wallet software. They're still optional, though.